coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
+  "result": {
+    "covered_percent": 94.15
+  }
+}
```

Comment:
Did you add these coverage files explicitly? The `.gitignore` file should be ignoring them.